src/sentry/middleware/ratelimit.py

```diff
@@ -44,17 +46,24 @@ def process_view(self, request: Request, view_func, view_args, view_kwargs) -> R
         if rate_limit is None:
             return
 
-        rate_limit_check_dict = above_rate_limit_check(key, rate_limit)
-        if rate_limit_check_dict["is_limited"]:
+        self.rate_limit_check_dict = above_rate_limit_check(key, rate_limit)
+        if self.rate_limit_check_dict["is_limited"]:
             request.will_be_rate_limited = True
             enforce_rate_limit = getattr(view_func.view_class, "enforce_rate_limit", False)
             if enforce_rate_limit:
                 return HttpResponse(
                     {
                         "detail": DEFAULT_ERROR_MESSAGE.format(
-                            limit=rate_limit_check_dict["limit"],
-                            window=rate_limit_check_dict["window"],
+                            limit=self.rate_limit_check_dict["limit"],
+                            window=self.rate_limit_check_dict["window"],
                         )
                     },
                     status=429,
                 )
+
+    def process_response(self, request: Request, response: Response) -> Response:
+        if self.rate_limit_check_dict is not None:
+            response["X-Sentry-Ratelimit-Count"] = self.rate_limit_check_dict.get("current", 0)
```

Comment:
I don't think it's possible for `current` to be missing.

Reply:
Pasting this here because I think the datadog headers are actually more intuitive

![image](https://user-images.githubusercontent.com/3169433/148468006-56dc0e38-ec1b-4669-a87d-1368206a6007.png)

Reply:
Pasting this here because I think the datadog headers are more intuitive

![image](https://user-images.githubusercontent.com/3169433/148468156-a0336aa9-0f7e-4813-b1e3-c699ad1a31f5.png)